src/RoleObjects/GridData.js

```diff
@@ -1,25 +1,42 @@
 import KeyCodes from 'keycodes-enum';
 import _ from 'lodash';
 import TableData from './TableData';
+import { ROLES } from '../Roles';
 
 export default class GridData extends TableData {
   constructor(displayObject, role, domIdPrefix) {
     super(displayObject, role, domIdPrefix);
-    _.bindAll(this, 'onKeyDown');
-    this._reactProps.onKeyDown = this.onKeyDown;
```

Comment:
The prior approach of making a new `onKeyDown` function instead of overriding the base class's `_onKeyDown` meant that the consuming app would lose WAI-ARIA behavior implemented in the subclass when the `enableKeyEvents` setter set to true.  The intent of `enableKeyEvents` is for the consuming app to do custom behavior (support more keystrokes, etc) not to replace.